js/components/TempTester.js

```diff
@@ -33,7 +36,36 @@ function mapStateToProps(state) {
 }
 
 const mapDispatchToProps = (dispatch) => {
+
+  function dispatchMessageFromRDP(packet) {
+    let {type} = packet;
+    if (type === "consoleAPICall") {
+      let message = prepareMessageInput("ConsoleGeneric", packet);
+      dispatch(messageAdd(message))
```

Comment:
it feels like `dispatch` could live outside of the `if`. Folks in React often use switch statement for this kind of thing... I'm ok with leaving it as lots of `if`s if you prefer that, though.